src/app/courses/courses.module.ts

```diff
@@ -16,10 +22,13 @@ import { SearchCourseComponent } from './components/search-course/search-course.
     CourseCardComponent,
     ModalWindowComponent,
     SearchCourseComponent,
+    DurationPipe,
```

Comment:
Pipes, directives, or other util functions should be imported with the Shared module.

Reply:
fixed thanks